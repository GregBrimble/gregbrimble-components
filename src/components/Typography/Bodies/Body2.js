import React from "react"

import styled from "styled-components"
import { Body } from "./shared"

const Body2 = styled(Body)`
  font-size: 0.875rem;
  line-height: 1.25rem;
`

export default Body2
