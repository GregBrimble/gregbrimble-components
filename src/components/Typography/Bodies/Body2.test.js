import React from "react"
import Body2 from "./Body2"
import renderer from "react-test-renderer"

describe("Body2", () => {
  it("renders properly", () => {
    const tree = renderer.create(<Body2>Body 2</Body2>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
