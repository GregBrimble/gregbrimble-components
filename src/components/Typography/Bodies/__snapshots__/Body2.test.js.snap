// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body2 renders properly 1`] = `
<p
  className="sc-bdVaJa sc-bwzfXH iJsFip"
>
  Body 2
</p>
`;
